tests/console/podman_image.pm

```diff
@@ -25,6 +25,17 @@ sub run {
 
     zypper_call "in podman";
 
+    # If there is no other CNI configured...
+    if (get_var("CNI", "podman") == "podman") {
```

Comment:
don't you mean this to be a check_var?
`
if checkvar("CNI", "podman")`

Reply:
No, that would do something else entirely.